libcst/codemod/visitors/_gather_unused_imports.py

```diff
@@ -37,9 +36,18 @@ class GatherUnusedImportsVisitor(ContextAwareVisitor):
         ScopeProvider,
     )
 
-    def __init__(self, context: CodemodContext) -> None:
+    def __init__(
+        self,
+        context: CodemodContext,
+        ignored_modules: Optional[Set[str]] = None,
+        typing_functions: Optional[Set[str]] = None,
+    ) -> None:
         super().__init__(context)
 
+        self._ignored_modules: Set[str] = (
+            ignored_modules if ignored_modules is not None else {"__future__"}
+        )
```

Comment:
```suggestion
    def __init__(
        self,
        context: CodemodContext,
        ignored_modules: Optional[Set[str]] = {"__future__"},
        typing_functions: Optional[Set[str]] = None,
    ) -> None:
        super().__init__(context)
        self._ignored_modules: Set[str] = ignored_modules
```

Reply:
I'll change these to `Collection`s too tomorrow, once I'm at my computer, and merge this PR afterwards.